astrobee/launch/robot/MLP.launch

```diff
@@ -111,16 +109,6 @@
       <arg name="debug" value="$(arg debug)" />
       <arg name="default" value="true" />
     </include>
-    <include file="$(find ff_util)/launch/ff_nodelet.launch">
```

Comment:
why did this go away?

Reply:
Oh I think you commented on a commit change, this is there in the final pr (I think I had accidentally removed it at some point)

Reply:
oh oops. great!